packages/fx-core/src/core/question.ts

```diff
@@ -55,6 +58,33 @@ export const QuestionRootFolder: FolderQuestion = {
   title: "Workspace folder",
 };
 
+export const QuestionSelectTargetEnvironment: SingleSelectQuestion = {
+  type: "singleSelect",
+  name: CoreQuestionNames.TargetEnvName,
+  title: "Select the target environment",
+  staticOptions: [],
+  skipSingleOption: true,
+  forgetLastValue: true,
```

Comment:
Why not make the `forgetLastValue` to be `false` to remember the last input value?

Reply:
If not, the display order of environment list might be changed and `+ new environment` won't be displayed as the 1st place, which would be weird.